README.md

```diff
@@ -25,13 +25,14 @@ In this challenge, create a web API around the following resources: `Projects` a
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] Mention two parts of Express that you learned about this week.
+`MiddleWare & server side routing`
 
 - [ ] Describe Middleware?
-
+    `Is this bodyguard right, that it verifies certain requirements that are need to be met, if requirements are not met, it does something, if the are, it enables the next middleware (if there's any) to be run`
```

Comment:
this can be one way to write middleware, as a auth check, middleware is any custom or third party functionality that you want to happen inbetween a server call or action and whats next or before